src/_pytest/capture.py

```diff
@@ -10,7 +10,10 @@
 from io import UnsupportedOperation
 from tempfile import TemporaryFile
 
-import pytest
+from .config import hookimpl
```

Comment:
curious why the imports changed, maybe I missed this

Reply:
Removes the import cycle to pytest

Else we can't import parts of pytest without breaking

Reply:
can you show me what you mean -- like stacktrace or reproduction or whatnot, I'm not sure I understand (and I don't understand why it's necessary for this PR)

Reply:
Pytest imports from all over the place this preveting isolation, if everything triggers every import then I can't let it effectively tell me what is going on

Reply:
Last stack trace I had if this disappeared a few months ago in a terminal, perhaps I could recreate it when I'm back on that computer